webapp/src/hidbridge.ts

```diff
@@ -182,7 +182,7 @@ export function initAsync() {
     return initPromise
         .then((w) => {
             wrapper = w;
-            if (pxt.winrt.isWinRT() && !isFirstInit) {
+            if (force || pxt.winrt.isWinRT() && !isFirstInit) {
```

Comment:
Why do we need a force argument?

Reply:
To handle the case where the device is plugged in and sending data, then unplugged, then plugged in again.  We want to reestablish the HID connection the second time it's plugged in, without having to reload the editor.

Reply:
We should revisit the logic to handle connection/disconnection of HID devices. THis also impact flashing.

Reply:
Added this TODO to https://github.com/Microsoft/pxt/issues/3125